config/crd/bases/kubeadm.bootstrap.cluster.sigs.k8s.io_kubeadmbootstrapconfigs.yaml

```diff
@@ -1060,9 +1060,6 @@ spec:
               type: string
             phase:
               description: Phase is the state of the KubeadmBootstrapConfig object
-              enum:
-              - Ready
-              - Unknown
               type: string
```

Comment:
Not sure if you need `Phase` here, IIRC we decided against Enums for the Phase field. `Status.Ready` is a required field though

Reply:
I thought the field was Phase and the value was Ready -- did that change to field is Ready and value is... true?

Reply:
Yeah, see https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/896/files#diff-e2e7d95efae5325d16854f456d43bcc3R85

Reply:
ok, i see, where did that decision take place? I may be not watching a space I should be watching

Reply:
I'll need to dig up some old CAPI PR, It was a while ago when we discussed with @ncdc @detiber @davidewatson and others about not using Phase, but instead switch to a single boolean field w.r.t. readiness. We've done the same upstream with `Status.InfrastructureReady` and `Status.BootstrapReady`.

This particular issue is a documentation one that should be addressed once we figure out the CAPI gitbook story

Reply:
i'm surprised the proposal wasn't updated -- is that pretty much out of date? will I have to dig through all the old CAPI PRs to find more decisions like this?

Reply:
I can open a PR to update the proposal, afaik other changes have been synced